content/tutorials/single-page-tutorials/setup_iota-js_ for_development.md

```diff
@@ -0,0 +1,178 @@
+# New Tutorial - How to setup iota.js for development
+
+> In this tutorial you will learn how to create a new project, generate a new address and receive testnet funds from the faucet.
+
+
+## Prerequisites
+
+- Development Environment and Libraries
+  - Installed [Node.js](https://nodejs.org/en/)
+
+***
+
+## Setup
+
+### Prepare development environment
+
+Create a new folder for the tutorial in a location of your choice and navigate to it:
+
+```bash
+mkdir iota-js-setup
+cd iota-js-setup
+```
+
+Run the Node.js initializer to configure the project:
+
+```bash
+npm init --yes
+```
+
+
+Install the iota.js library and other helper dependencies:
+> **NOTE:**  Installation might take a while
+```bash
+npm install @iota/iota.js@next
+npm install bip39 dotenv
+```
+
+
+Now create a new file `.env` and add the content below:
+
+```javascript
+MNEMONIC = "<You_will_create_your_own_mnemonic_seed_phrase_in_the_next_step_and_paste_it_in_here>"
+```
+
+Enter your desired account name as well as a secure Stronghold password. Your new mnemonic seed phrase will be created in the next step and you will paste it in here afterwards.
+
+***
+
+
+## Create mnemonic script
+
+Create a new file `create-mnemonic.js` and add the following code:
+
+```javascript
+// Libraries
+const bip39 = require('bip39')
+ 
+// For better readability, some console output will be printed in a different color
+const consoleColor = '\x1b[36m%s\x1b[0m';
+async function run() {
+	// A 256-bits entropy leads to a 24 word mnemonic seed phrase
+	const mnemonic = bip39.generateMnemonic(256);
+	console.log(consoleColor, 'Copy your mnemonic seed phrase and paste it into the .env file:');
+	console.log(mnemonic, '\n');
+}
+run();
+```
+
+Run the script `create-mnemonic.js` and check the console output for your new mnemonic seed phrase:
+```console
+node create-mnemonic.js
+```
+Now you can copy the seed phrase and paste it into the previously created `.env` file.
+
+***
+
+Your `package.json` file should look something like this:
+
+```yaml
+{
+  "name": "iota-js-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@iota/iota.js": "^1.9.0-stardust.25",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+***
```

Comment:
Shouldn't this be moved to the beginning after installing the devs?

Reply:
It's just an example how it should look like after the installation part.

Reply:
Sure I got that. I was just surprised reading it after already having done the installation some time ago 🤔

Reply:
I must say I agree with @Dr-Electron , this seems awfully out of place here as we're in the generate mnemonic section by now.

Reply:
Fixed!